tool/enable-mkmf-verbose

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -eu
+
+set -x
+
+gem_home="$(gem environment home)"
+mkmf_rb_path="$(find "${gem_home}"/../.. -name mkmf.rb)"
+
+# Backup the original file.
+cp -p "${mkmf_rb_path}" "${mkmf_rb_path}.orig"
+
+# Replace the target line to set the verbose option.
+sed -i -e 's/^V = .*/V = 1/' "${mkmf_rb_path}"
+
+# Print the difference between the original and modified file.
+diff "${mkmf_rb_path}.orig" "${mkmf_rb_path}" || :
```

Comment:
Note the `diff` command is to notice and debug easily when the `mkmf.rb` may change in the future.